### DISCUSSION TITLE
How i can add V2 to project?

### DISCUSSION THREAD

**Discussion by shurshilov** [KevinVandy/mantine-react-table#385] [Q&A] [answered]
I use vite and on npm only V1 available.
How can i try V2?

**Comment by rommelmamedov** [comment, marked as answer]
Just update the version like ? 🤔:

```json 
 "mantine-react-table": "v2.0.0-beta.6"
 ```

<img width="1331" alt="image" src="https://github.com/user-attachments/assets/2acbf4d6-ab87-491c-a7e4-c6c87fd77785">

**Reply by shurshilov** [reply]
Thanks, but where you found v2.0.0-beta.6 version?)))

**Reply by shurshilov** [reply]
damn strange, I just couldn't find her in npm, now I see, sorry bro, it's time to rest))

**Reply by shurshilov** [reply]
I just opened this page and I think damn there is no release in npm) but I didn't think to check the versions) so it seemed that you didn't release them)
![image](https://github.com/user-attachments/assets/600a0635-0b7c-4d83-b944-801c024951d5)

**Reply by KevinVandy** [reply]
V2 is just still in beta
